### PR TITLE
feat: prettier and fix linting errors #147

### LINKED ISSUE #147: Add prettier to eslint config
# Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked to make sure that this issue has not already been filed

# Problem

- Our client-side code formatting is inconsistent.

# Solution 
- We can use prettier along with eslint to handle formatting.

### PR DESCRIPTION
**Pull request by kartikpandey2**
# Include
- Added prettier to handle code formatting.
- Removed some esLint rules which conflict with prettier 
- Added parser option to eslint config to  [fix parsing error](https://stackoverflow.com/questions/36001552/eslint-parsing-error-unexpected-token)
- Fixed linting errors and formatted code with prettier
- Added husky hooks before git commit and git push

**Closing issues**

Put closes #147

### CODE REVIEW

**Comment by root5533** [comment]
All look good. Please update PR and fix merge conflict @kartikpandey2 .

**Comment by kartikpandey2** [comment]
@root5533  done, please review now 😃

**Comment by kartikpandey2** [comment]
@root5533  what is the issue .? 😅

**Comment by root5533** [comment]
@kartikpandey2 I don't think you have rebased the master branch properly. I couldn't rebase after I approved. Have you updated with the latest commits in your local branch ?

**Comment by kartikpandey2** [comment]
I believe I have. Let me try again. @root5533

**Comment by kartikpandey2** [comment]
@root5533  Yes, I had not rebase correctly. Now corrected, please review now. 😃
